src/studio-home/processing-courses/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Stack } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { getStudioHomeData } from '../data/selectors';
+import CourseItem from './course-item';
+import messages from './messages';
+
+const ProcessingCourses = () => {
+  const intl = useIntl();
+  const { inProcessCourseActions } = useSelector(getStudioHomeData);
+
+  return (
+    <div>
```

Comment:
Maybe with fragment will be better
```suggestion
    <></>
```

Reply:
yeah, sure!